local-deployment/docker-compose.yml

```diff
@@ -0,0 +1,481 @@
+    volumes:
+      - ${DOCKER_SHARED_VOLUME}:/asn1_codec_share
+    logging:
+      options:
+        max-size: "10m"
+        max-file: "5"
+    restart: on-failure
+  
+  aem:
+    image: usdotjpoode/asn1_codec:2024-q2
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      ACM_CONFIG_FILE: aem.properties
+      ACM_LOG_TO_CONSOLE: "true"
+      ACM_LOG_TO_FILE: "false"
+      ACM_LOG_LEVEL: ERROR
+    depends_on:
+      - kafka
+    volumes:
+      - ${DOCKER_SHARED_VOLUME}:/asn1_codec_share
+    logging:
+      options:
+        max-size: "10m"
+        max-file: "5"
+    restart: on-failure
+
+  # Note: the SDX does not accept unsigned data, so this is commented out for now
+  # sdw_depositor:
+  #   image: usdotjpoode/jpo-sdw-depositor:2024-q2
+  #   environment:
+  #     DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+  #     SDW_EMAIL_LIST: test@gmail.com
+  #     SDW_EMAIL_FROM: test@gmail.com
+  #     SDW_API_KEY: ${SDW_API_KEY}
+  #     SDW_DESTINATION_URL: ${SDW_REST_URL}
+  #     SPRING_MAIL_HOST: 172.0.0.1
+  #     SPRING_MAIL_PORT: 123
+  #     SDW_SUBSCRIPTION_TOPIC: topic.SDWDepositorInput
+  #   depends_on:
+  #    - kafka
+  #    - ode
+  #   logging:
+  #     options:
+  #       max-size: "10m" 
+  #       max-file: "5"
+  
+  postgres:
+    image: postgis/postgis:15-master
+    restart: always
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+    volumes:
+      - pgdb:/var/lib/postgresql/data
+    logging:
+      options:
+        max-size: '10m'
+  
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${MONGO_USER}
+      MONGO_INITDB_ROOT_PASSWORD: ${MONGO_PASSWORD}
+    ports:
+      - '27017:27017'
+    volumes:
+      - mongodb_data_container:/data/db
+    logging:
+      options:
+        max-size: '10m'
+  # end of external dependencies ----------------------------------------------
+
+  # internal dependencies ------------------------------------------------------
+  milepost-graph-db:
+    build: ../milepost-graph-db
+    restart: always
+    ports:
+      - "6474:7474"
+      - "6473:7473"
+      - "6687:7687"
+    volumes:
+      - ../milepost-graph-db/neo-data/plugins:/var/lib/neo4j/plugins
+      - ../milepost-graph-db/neo-data/import:/var/lib/neo4j/import
+      - ../milepost-graph-db/neo-data/conf:/var/lib/neo4j/conf
+      - ../milepost-graph-db/neo-data/data:/var/lib/neo4j/data
+    environment:
+      NEO4J_AUTH: none
+    logging:
+      options:
+        max-size: "20m"
+        max-file: "3"
+
+  kafka-connect:
+    image: quay.io/debezium/connect:2.7
+    restart: always
+    ports:
+      - "8083:8083"
+    environment:
+      BOOTSTRAP_SERVERS: kafka:9094
+      GROUP_ID: 1
+      CONFIG_STORAGE_TOPIC: my_connect_configs
+      OFFSET_STORAGE_TOPIC: my_connect_offsets
+      STATUS_STORAGE_TOPIC: my_connect_statuses
+    depends_on:
+      - kafka
+      - postgres
```

Comment:
The `kafka-connect` service can be removed as it was only used for the ad-hoc listener.